go.sum

```diff
@@ -767,6 +767,9 @@ istio.io/operator v0.0.0-20190813061052-3b21fa19e418/go.mod h1:WudeDwe5Tdj+6dSLi
 istio.io/pkg v0.0.0-20190515193414-9332430ad747/go.mod h1:0EkPwmR0tESYjN4Ilq1D52nTBurXaQvny3r2VY4j4tw=
 istio.io/pkg v0.0.0-20190731230704-fcbac27d69d5 h1:HcASpvj/fuuABkYH9YbsTGEOT75YHyWvvFnTe229zXs=
 istio.io/pkg v0.0.0-20190731230704-fcbac27d69d5/go.mod h1:We4ZQuCbiiNfXge2GfOshBsyDXVwzFwP8703V5DcM14=
+istio.io/pkg v0.0.0-20190737041230-fcbac27d69d5/go.mod h1:We4ZQuCbiiNfXge2GfOshBsyDXVwzFwP8703V5DcM14=
+istio.io/pkg v0.0.0-20190813222952-67b4f108fe0e h1:8RI4/zsMtrjkvyUqvy5CTGyVQPL86rERoWdm1npYU2A=
+istio.io/pkg v0.0.0-20190813222952-67b4f108fe0e/go.mod h1:We4ZQuCbiiNfXge2GfOshBsyDXVwzFwP8703V5DcM14=
```

Comment:
can you run mod tidy and mod vendor?

see https://github.com/istio/istio/wiki/Vendor-FAQ

Reply:
sure, thanks for letting me know